serious_django_permissions/helpers.py

```diff
@@ -1,8 +1,50 @@
 import re
+from abc import ABC
+import importlib
+import inspect
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+from .permissions import Permission
+from .groups import Group
 
 # https://stackoverflow.com/a/1176023/3090225
 first_cap_re = re.compile('(.)([A-Z][a-z]+)')
 all_cap_re = re.compile('([a-z0-9])([A-Z])')
 def camel_to_snake(name):
     s1 = first_cap_re.sub(r'\1_\2', name)
     return all_cap_re.sub(r'\1_\2', s1).lower()
+
+
+def create_permissions(*args, **options):
+    for app in settings.INSTALLED_APPS:
+            lib = importlib.util.find_spec("{}.permissions".format(app))
+            if lib:
+                lib = importlib.import_module("{}.permissions".format(app))
+                for name, obj in inspect.getmembers(lib):
+                    if type(obj) == type(Permission) and issubclass(obj, Permission)\
+                       and obj.__base__ != ABC:
+                        perm, created_at = obj.get_or_create()
+
+def create_groups(*args, **options):
+    create_permissions()
+
+    if not getattr(settings, 'DEFAULT_GROUPS_MODULE', None):
+        raise AttributeError("DEFAULT_GROUPS_MODULE setting is not set!")
+
+    else:
+        groups_module = settings.DEFAULT_GROUPS_MODULE
+
+    lib = importlib.import_module(groups_module)
+    for name, obj in inspect.getmembers(lib):
+        if type(obj) == type(Group) and issubclass(obj, Group)\
+           and obj.__base__ != ABC:
+            group, created_at = obj.get_or_create()
+            perm_db_objs = [
+                perm.get_or_create()[0] for perm in obj.permissions
+            ]
+            group.permissions.set(perm_db_objs)
+
+def setup_permissions():
```

Comment:
what's that for?

Reply:
I guess I see, but `setup_permissions` is really the most confusing option out of these, as it calls `create_groups`, and `create_permissions` also exists. Why not name the helper functions `create_permissions` and `create_groups_and_permissions` and scrap the third function which is just an alias?

Reply:
i.e. don't have any `setup_` function